README.md

```diff
@@ -64,13 +83,18 @@ optional arguments:
 ```
 
 ### Phases
-* The prepare phase builds TPC-H dbgen and querygen and creates the load and update files. 
-* The load phase cleans the database (if required), loads the tables into the database and 
+* `prepare`
+  * The prepare phase builds TPC-H dbgen and querygen and creates the load and update files. 
```

Comment:
I don't think you need to add a separate bullet for this. If you wanted a new line something like would be better I think.

Markdown:
```
* **`prepare`**(this line has two blank spaces at the end)  
    The prepare phase builds TPC-H dbgen and querygen and creates the load and update files.
```
Output:
* **`prepare`**(this line has two blank spaces at the end)  
    The prepare phase builds TPC-H dbgen and querygen and creates the load and update files.